docker-compose.yml

```diff
@@ -119,6 +120,7 @@ services:
         source: config/prometheus.yml
         target: /etc/prometheus/prometheus.yml
       - prom_data:/prometheus
+      - /var/run/docker.sock:/var/run/docker.sock:ro
```

Comment:
Hmm. Why does this need access to docker? Does this let prometheus scrape metrics on docker itself?